NOnion/Cells/Relay/CellPlainRelay.fs

```diff
@@ -16,7 +16,7 @@ type RelayData =
     | RelayExtend
     | RelayExtended
     | RelayTruncate
-    | RelayTruncated
+    | RelayTruncated of Reason: byte
```

Comment:
and I know it's not part of this diff, but man, I just saw in this file a failwith with "" as the error msg, omg, let's fix that